greenboot-script-runner.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+run_required_scripts () {
+  required_greenboot_scripts=`ls /etc/greenboot.d/required/*.sh`
+  set +e
+  for script in $required_greenboot_scripts; do
+    bash $script
+    if [ $? -ne 0 ]; then
+      echo "FAILURE (return value '$?')" >&2 | systemd-cat -t "greenboot Script: '$script'" -p emerg
```

Comment:
let's call this "exit code" rather than "return value". Also why redirect to STDERR (`>&2`) here ?

Reply:
is "return code" (rc) ok?

Reply:
nvmd. exit code it is.